XenOvfTransport/Import.cs

```diff
@@ -1293,6 +1295,12 @@ private XenRef<VM> DefineSystem(Session xenSession, VirtualHardwareSection_Type
 	                    case "nvram":
 		                    newVm.NVRAM = SplitStringIntoDictionary(xcsd.Value.Value);
 		                    break;
+                        case "vgpu":

```

Comment:
Adding the vgpus to the hashtable is not needed here, because they are not going to be imported from the hashtable directly (we process them explicitly, in `#region Set vgpu` above)

Reply:
yes,  As the `UpdateFrom` match the key `VGPUs`, not `vgpu`,
I add vgpus to hashtable just to keep the original behavior (add vgpus to the hashtable),
But it make sense to skip vGPUs here. I will update it.